hub/core/version_control/dataset_diff.py

```diff
@@ -25,6 +54,26 @@ def frombuffer(cls, data: bytes) -> "DatasetDiff":
         """Creates a DatasetDiff object from bytes"""
         dataset_diff = cls()
         dataset_diff.info_updated = bool(int.from_bytes(data[:1], "big"))
+        len_renamed = int.from_bytes(data[1:9], "big")
```

Comment:
wouldn't this lead to issues with backwards compatibility?

Reply:
we should be good since we are taking slices of `data`. `len_renamed` and `len_deleted` will just be 0.